config/locales/en/en.yml

```diff
@@ -329,7 +329,7 @@ en:
   quickbooks:
     authorization_failure: "You must be connected to QuickBooks before you can access transactions. Please go to %{settings} to connect."
     class_location_notice_html: "<b>NOTE:</b> Before you can add transactions, you need to make sure <b>Classes</b> and <b>Locations</b> (a.k.a. Divisions) are enabled in your QuickBooks company settings."
-    full_sync_required: "A full sync is required before you can access transactions. Please go to %{settings} to sync."
+    full_reset_required: "A full reset is required before you can access transactions. Please go to %{settings} to sync."
```

Comment:
How about `A Quickbooks data reset is required before you can access transactions. Please go to %{settings} to perform a reset.`